indexer-opts/README.md

```diff
@@ -0,0 +1,52 @@
+# indexer-opts - Opts struct for CLI arguments for microindexers
+
+This lib is a shared `clap::Parser` instance that defines CLI arguments for all microindexers.
+
+### Features:
+
+- Basic arguments for indexer to start
+- `init_tracing` for initializing the logs for indexer
+- `StartMode` that allows to choose the strategy for restarts:
+  - `from-latest`
+  - `from-interruption` (default)
+
+### Parameters
+
+- `indexer-id` | **Required** Sets the micro-indexer instance ID (for reading/writing indexer meta-data)
+- `start-block-height` | **Required** Block height to start the stream from (not used if `start_mode == from-latest`)
```

Comment:
Look, we have 2 main goals:
- start indexer from some point (first start)
- continue from interruption

Both goals are simultaneous. I just want to remind you how we used to achieve it:

1. You start indexer "from-block" 1
2. Then you stop it
3. Start it "from-interruption" in order for service to work

The "from interruption" command is written on the server for restarting.

Reply:
I suggest to get rid of `--start-block-height`
Let's have enum instead of existing `--start-mode`:
- `--sync-latest` with no arguments provided
- `--sync-from-interruption` with block_height as the agrument

I don't insist on exact wording, it definitely can be better

Reply:
I am trying to achieve the way of starting the indexer with `.env` only.

Thus the run script would require only `/path/to/bin/indexer` and all other settings would be provided via `.env` file. 
I don't know how to make Enum with parameters parsable by `env` derive, and I am not sure it is even possible.